client/src/components/Navbar.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import logoImage from './../images'
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem, Button } from 'reactstrap';
+
+/**
+ * Container for the navbar component. 
+ * Currently shows a basic navbar with dummy links since there are no pages.
+ */
+class BFSNav extends Component {
```

Comment:
BFSNav --> PrimaryNavBar or something else. Don't need to rename it though, the end user won't really see this or understand what it means